### DISCUSSION TITLE
Feature Request: Digests in Smart Delivery

### DISCUSSION THREAD

**Discussion by standingdreams** [magicbell/community#197] [General]
It would be great to be able to have a digest of notifications that a user received over a specific time frame. This would allow a user to receive a weekly summary of all notifications a user may have received that week. Is this something that's possible with Smart Delivery or planned in the future?

**Comment by pianomansam** [comment]
Adding my support for this as well. Somewhere I heard that this was going to happen. Other services provide it. It'd be awesome if MagicBell did too

**Comment by unamashana** [comment]
Hey @pianomansam & @standingdreams we have this feature available in our new API version. If you ping me on Slack, we can get you access to try it out and launch with us! - https://magicbell.to/slack
